pkg/namespace/controllers/reconciler.go

```diff
@@ -68,6 +68,9 @@ func (c *Reconciler) getSliceNameFromNs(ns string) (string, error) {
 }
 
 func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+
+	excludedNsEnv := utils.GetEnvOrDefault("EXCLUDED_NS", "kube-system,default,kubeslice-system,kube-node-lease,kube-public,istio-system")
```

Comment:
Can we have a global var defaultExcludedNS with all the namespaces we want to exclude? then we could do: utils.GetEnvOrDefault("EXCLUDED_NS", strings.Join(defaultExcludedNs, ","))

Reply:
made the change.
strings.Join and split would be unnecessary so just using comma separated string in the global const.